crates/vm/wasm/src/gatekeeper.rs

```diff
@@ -749,3 +749,73 @@ impl FunctionMiddleware for FunctionGatekeeper {
         }
     }
 }
+
```

Comment:
Can you always add this "separator" comment before the `tests` module? I have this for all other tests:

```rust
// ----------------------------------- test ------------------------------------
```